.github/workflows/build.yaml

```diff
@@ -26,9 +24,13 @@ jobs:
         with:
           poetry-version: 1.8.3
       - name: Build the package
-        run: poetry build 
+        run: poetry build
       - name: Publish package distributions to PyPI
         uses: pypa/gh-action-pypi-publish@release/v1
+        with:
+          repository-url: https://test.pypi.org/project/qref/ # TODO: change to https://pypi.org/project/qref  after testing
+      - name: Install mike
+        run: poetry install --with docs
       - name: Build docs
         run: mike deploy $RELEASE_VERSION  latest --update-aliases
```

Comment:
Maybe we should use:
```
poetry run mkdocs build
poetry run mike deploy [version] -u
```

Reply:
I think you're right - we were running into some issues with how GH actions handles virtual envs with poetry and `poetry run` is definitely safer.
Not sure about the `[version] -u` part though, whatever works 🤷‍♂️

Reply:
https://github.com/PsiQ/qref/pull/59/commits#:~:text=Adding%20poetry%20run%20instead%20of%20mike%20deploy